created section "Reviews", added html, all styles (mob, tablet, desktop), images 1x and 2x

Roman-Nazymko opened the pull request.

Roman-Nazymko committed:
added css and html

Roman-Nazymko committed:
Merge branch 'main' into html-css/reviews

Bignichok commented on src/partials/reviewers.html:
```diff
@@ -1 +1,103 @@
+      their experiences, progress, and the transformative impact
+    </p>
+    <div
+      class="reviews-box wow animate-animated animate-bounceInUp"
```
`wow animate-animated animate-bounceInUp` why do you need this classes?

Roman-Nazymko replied:
Зараз видалю. Грався з анімацією, але не вийшло. Забув видалити.

Bignichok commented on src/partials/reviewers.html:
```diff
@@ -1 +1,103 @@
+    </p>
+    <div
+      class="reviews-box wow animate-animated animate-bounceInUp"
+      data-wow-duration="1s"
+      data-wow-delay="0.3s"
```
and this data attributes?

Roman-Nazymko replied:
Видалив

Bignichok commented on src/css/reviewers.css:
```diff
@@ -1,3 +1,196 @@
+  margin: 0 auto;
+}
+
+@media screen and (min-width: 375px) {
```
this media query does not make sense

Roman-Nazymko replied:
<img width="820" alt="Знімок екрана 2024-10-03 о 16 44 51" src="https://github.com/user-attachments/assets/fd9418c9-8f62-40a4-8e29-dbcfba6a368f">
Як же не має сенсу? Якщо у фігмі стоїть розмір 275 px. А в мене розширюється до 295 px.Можу відразу поставити замість width: 100%, max- width: 275 px

Bignichok replied:
ми використовуємо тільки 2 768px і 1280px брекпоінта, якщо тобі потібен додатковий, отже ти щось робиш не правильно
те що воно тягнеться і ширина буде не як на макеті то це не страшно, її не потрібно обмежувати

Roman-Nazymko replied:
Чи напевно просто таких мобілок не існує?

Bignichok replied:
![Uploading image.png…]()

Bignichok replied:
![image](https://github.com/user-attachments/assets/1ece4874-6bf3-4f60-9105-79aefd758651)

Bignichok commented on src/css/reviewers.css:
```diff
@@ -1,3 +1,196 @@
+    letter-spacing: -0.8px;
+  }
+}
+.reviewers-underlined-text {
+  position: relative;
+}
+
+.reviewers-underlined-text::after {
+  position: absolute;
+  bottom: 3px;
+  left: 0;
+  content: '';
+  z-index: -1;
+  width: 100%;
+  height: 12px;
+  border-radius: 8px;
+  background-color: var(--underline-word);
+}
+
+@media screen and (min-width: 768px) {
+  .reviewers-underlined-text::after {
+    height: 20px;
+  }
+}
```
this code is redundant, reuse our class `custom-underline`

Roman-Nazymko replied:
Зараз зроблю

Bignichok commented on src/css/reviewers.css:
```diff
@@ -1,3 +1,196 @@
+  padding: 20px 20px 40px 20px;
+}
+
+@media screen and (min-width: 375px) {
```
does not make sense

Roman-Nazymko replied:
Видалив

Roman-Nazymko committed:
fixed bags

Deleted class for animation. Deleted medias (min 375px). Changed class for underline line on custom-underline and deleted styles for before class

Roman-Nazymko committed:
Merge branch 'main' into features/reviews-section
